src/Phansible/Controller/BundleController.php

```diff
@@ -124,4 +123,31 @@ protected function outputBundle($zipPath, Application $app, $filename)
 
         return $response;
     }
+
+    /**
+     * @return \Phansible\Model\VagrantBundle
+     */
+    private function getVagrantBundle()
```

Comment:
Do you have any tips to make this work?

I don't know if the Flint Framework have any support for this feature.